### PR TITLE
add the proposal doc for device management interface (DMI)

### PR DESCRIPTION
**Pull request by RyanZhaoXB**
Signed-off-by: RyanZhaoXB <zhaoran11@huawei.com>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #https://github.com/kubeedge/kubeedge/issues/3760

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by RyanZhaoXB** [comment]
@TianTianBigWang hello, this pr proposed the document of dmi. please help to review it.

**Comment by TianTianBigWang** [comment]
lgtm

File: docs/proposals/device-management-interface.md [hunk 1]
```diff
@@ -0,0 +1,364 @@
+## KubeEdge Device Management Interface（DMI）
+
+### Goal
+
+This proposal provides the Device Management Interface（i.e. DMI） which integrates the device management interface and optimizes the device management capability of KubeEdge. It is the goal of KubeEdge DMI that builds a cloud native device twin management platform which covers device management, device operation and device data.
```
[the remark this answers is not in the document]
Comment (RyanZhaoXB) [reply docs/proposals/device-management-interface.md]: fixed it

File: docs/proposals/device-management-interface.md [hunk 2]
```diff
@@ -0,0 +1,364 @@
+## KubeEdge Device Management Interface（DMI）
```
[the remark this answers is not in the document]
Comment (RyanZhaoXB) [reply docs/proposals/device-management-interface.md]: done

**Pull request closed without merging**